static/app/components/replays/breadcrumbs/replayTimelineTooltip.tsx

```diff
@@ -0,0 +1,111 @@
+import {Fragment, useCallback, useEffect, useRef, useState} from 'react';
+import {createPortal} from 'react-dom';
+import styled from '@emotion/styled';
+import {AnimatePresence} from 'framer-motion';
+
+import {Overlay} from 'sentry/components/overlay';
+import {space} from 'sentry/styles/space';
+
+const TOOLTIP_OFFSET = 10;
+
+type Props = {
+  /**
+   * Function used to compute the text of the cursor tooltip. Recieves the %
+   * value the cursor is within the container.
+   */
```

Comment:
What is the function here referring to?

Reply:
oops this was copied over and changed, I updated the comment!